packages/dds/matrix/src/matrix.ts

```diff
@@ -697,16 +907,36 @@ export class SharedMatrix<T = any>
 		for (const rowHandle of rowHandles) {
 			this.cells.clearRows(/* rowStart: */ rowHandle, /* rowCount: */ 1);
 			this.pending.clearRows(/* rowStart: */ rowHandle, /* rowCount: */ 1);
+			this.cellLastWriteTracker.clearRows(/* rowStart: */ rowHandle, /* rowCount: */ 1);
 		}
 	};
 
 	private readonly onColHandlesRecycled = (colHandles: Handle[]) => {
 		for (const colHandle of colHandles) {
 			this.cells.clearCols(/* colStart: */ colHandle, /* colCount: */ 1);
 			this.pending.clearCols(/* colStart: */ colHandle, /* colCount: */ 1);
+			this.cellLastWriteTracker.clearCols(/* colStart: */ colHandle, /* colCount: */ 1);
 		}
 	};
 
+	/**
+	 * Api to switch Set Op policy from Last Writer Win to First Writer Win. It only switches from LWW to FWW
+	 * and not from FWW to LWW. An op will be sent to communicate this to other clients.
+	 */
+	public switchSetCellPolicy() {
```

Comment:
@jatgarg, the name is not great, as it does not reflect what is switch to what. Should we change it to be more telling?